workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -1,90 +1,150 @@
 # Interfaces and Networks
 
-An `interface` defines a virtual network interface of a virtual machine.
+Connecting a virtual machine to a network consists of two parts. First,
+networks are specified in `spec.networks`. Then, interfaces backed by the
+networks are added to the VM by specifying them in
+`spec.domain.devices.interfaces`.
 
-A `network` specifies the backend of an `interface` and controls to which logical or physical device it is connected to.
+Each interface must have a corresponding network with the same name.
 
-Often there is more than one way to connect an `interface` to a `network`. A user can control this connection by choosing between different options.
-However, at time of writing this document there is only a single way to connect interfaces and pods, and this method is `delegateIp`.
+An `interface` defines a virtual network interface of a virtual machine (also
+called a frontend). A `network` specifies the backend of an `interface` and
+declares which logical or physical device it is connected to (also called as
+backend).
 
-## Default network connectivity of a virtual machine
+There are multiple ways of configuring an `interface` as well as a `network`.
 
-> **Note:** Currently, only a single interface is supported, and this must be connected to the pod network.
+All possible configuration options are available in the
+[Interface API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_interface)
+and [Network API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network).
 
-By default - thus if nothing else is specified - a virtual machine will get a single network interface connected to the pod network.
+> **Note:** Currently, the only supported network type is `pod`.
 
-## Customizing network interfaces and networks
+## Backend
 
-In order customize the network connectivity of a virtual machine, users must provide
+Network backends are configured in `spec.networks`. Each network must have a
+unique name. Additional fields declare which logical or physical device the
+network relates to.
 
-- a list of interfaces and
-- a list of networks each of the interfaces is connected to
+Each network definition can be referred to in no more than a single interface
+per virtual machine. If you need to connect the same virtual machine twice to
+the same physical or logical network, declare two networks with the same
+properties but unique names.
```

Comment:
Perhaps, we shouldn't talk about connecting the same network to multiple interfaces now. After all, it won't work now with the pod network delegate ip approach, not sure what's the benefit of it with slirp.
I think we should get back to this point later when we will have additional networks

Reply:
I thought "**Note:** Currently, the only supported network type is `pod`." gives enough of a signal it's not yet possible but intended to be. I will clean it up though.